lang/java/compiler/pom.xml

```diff
@@ -228,6 +228,11 @@
       <!-- can only be used from within ant -->
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <version>${mockito.version}</version>
```

Comment:
Does it need `<scope>test</scope>`? I didn't find a dependencyManagement in the parent poms for it.

Reply:
Yes, I'll limit the Mockito within test scope.